src/Microsoft.Identity.Web/Constants/ErrorMessage.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Web
+{
+    internal static class ErrorMessage
```

Comment:
>ErrorMessage [](start = 26, length = 12)

A couple of late messages for consideration in a future PR

Reply:
sounds good. thanks, henry